### ISSUE TITLE
About running time comparison with dcnv3

### ISSUE THREAD

**Issue by qiuzidian** [OpenGVLab/DCNv4#32]
Amazing engineering implementation! But i have some questions, I compared DCNv3 and DCNv4 in RTX2080ti, PyTorch 1.13.1 environment. DCNv3 uses the built-in PyTorch function and the measured results are as follows. Does DCNV4 not have a speed advantage when there are fewer channels and higher resolution? May I ask what the possible reason is?

float32
dcnv3: 2.234679698944092
dcnv4: 2.8860323429107666
float16
dcnv3: 1.4734604358673096
dcnv4: 2.88419246673584

the code are
```python
block1 = DCNv3(64, 64, group=4).cuda().eval()
block2 = DCNv4(64, 64, group=4).cuda().eval()
a = torch.rand(4, 64, 1920//4, 1088//4).cuda()
offset = torch.rand(4, 64, 1920//4, 1088//4).cuda()

for i in range(10):
    with torch.no_grad():
        block1(a, offset)
        block2(a, offset)

torch.cuda.synchronize()
start_time = time.time()
for i in range(100):
    with torch.no_grad():
        block1(a, offset)
torch.cuda.synchronize()
print('dcnv3:', time.time() - start_time)

torch.cuda.synchronize()
start_time = time.time()
for i in range(100):
    with torch.no_grad():
        block2(a, offset)
torch.cuda.synchronize()
print('dcnv4:', time.time() - start_time)
```

**Comment by zdk258** [comment]
So what is the reason?

**Comment by mdfareed92** [comment]
what is the reason?
